lib/provisioner/worker/plugins/automators/docker_automator/docker_automator.json

```diff
@@ -16,9 +16,15 @@
           },
           "publish_ports": {
             "label": "Container ports to publish",
-            "override": false,
+            "override": true,
             "type": "text",
             "tip": "Comma-separated list of container ports to publish"
+          },
+          "command": {
+            "label": "Command to run, including arguments",
```

Comment:
I just misunderstood as I thought this was a catchall to provide additional ARGs to docker.  Minor suggestion to help clear up this confusion would be to change label to something like "Docker COMMAND option, including arguments".  Otherwise 👍 lgtm